src/app/dashboard/dashboard.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { DashboardComponent } from './dashboard.component';
+
+describe('DashboardComponent', () => {
+  let component: DashboardComponent;
+  let fixture: ComponentFixture<DashboardComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [ DashboardComponent ]
```

Comment:
Add on line 12 `imports: [HttpClientTestingModule]` in order to fix the failing test.

Reply:
`RouterTestingModule` instead of `HttpClientTestingModule`.